src/utils/mutex-utils.js

```diff
@@ -35,10 +35,10 @@ const mockUnlock = (siteName) => {
   mockMutexObj[siteName] = false
 }
 
-const lock = async (siteName) => {
+const lock = async (siteName, lockLengthSeconds = 60) => {
   try {
     const ONE_MIN_FROM_CURR_DATE_IN_SECONDS_FROM_EPOCH_TIME =
```

Comment:
nit: the "one min" isn't true anymore, might want to rename the variable to avoid confusion?

Reply:
Fixed in https://github.com/isomerpages/isomercms-backend/pull/1149/commits/0cc6344c3cb654f3f7fd5dc3b009b514ccf9aa23!